### DISCUSSION TITLE
Squawk code is not working

### DISCUSSION THREAD

**Discussion by esc90109** [flybywiresim/aircraft#4094] [Help]
Hi, i´ve got the A32nx (Unstable Master Branch Build, updated today) and I can't enter a sqwak code into the TCAS panel. All butons on this panel are working. Do you have an answer? 
Thank you :)

**Comment by Paytheon** [comment]
The go around is to clear out the value in the TCAS first and then enter the sqwak code.

**Comment by tracernz** [comment]
You need to clear first (double press to clear whole code). That is how the real panel works.
